doc/experimental/phase1.md

```diff
@@ -21,203 +21,7 @@ Follow these [instructions](./disable-verrazzano.md) to disable the Verrazzano c
- 
-kubectl rollout restart deployment -n verrazzano-ingress-nginx ingress-controller-ingress-nginx-defaultbackend
-kubectl rollout status deployment -n verrazzano-ingress-nginx ingress-controller-ingress-nginx-defaultbackend -w
- 
-kubectl rollout restart deployment -n mysql-operator mysql-operator
-kubectl rollout status deployment -n mysql-operator mysql-operator -w
- 
-kubectl rollout restart deployment -n keycloak mysql-router
-kubectl rollout status deployment -n keycloak mysql-router -w
- 
-kubectl rollout restart statefulset -n keycloak mysql
-kubectl rollout status statefulset -n keycloak mysql -w
- 
-kubectl rollout restart statefulset -n keycloak keycloak
-kubectl rollout status statefulset -n keycloak keycloak -w
- 
-kubectl rollout restart statefulset -n verrazzano-monitoring prometheus-prometheus-operator-kube-p-prometheus
-kubectl rollout status statefulset -n verrazzano-monitoring prometheus-prometheus-operator-kube-p-prometheus -w
- 
-kubectl rollout restart statefulset -n verrazzano-system vmi-system-es-master
-kubectl rollout status statefulset -n verrazzano-system vmi-system-es-master -w
- 
-kubectl rollout restart deployment -n verrazzano-system vmi-system-es-ingest
-kubectl rollout status deployment -n verrazzano-system vmi-system-es-ingest -w
- 
-kubectl rollout restart deployment -n verrazzano-system vmi-system-osd
-kubectl rollout status deployment -n verrazzano-system vmi-system-osd -w
- 
-kubectl rollout restart deployment -n verrazzano-system vmi-system-es-data-0
-kubectl rollout status deployment -n verrazzano-system vmi-system-es-data-0 -w
-
-kubectl rollout restart deployment -n verrazzano-system vmi-system-es-data-1
-kubectl rollout status deployment -n verrazzano-system vmi-system-es-data-1 -w
-
-kubectl rollout restart deployment -n verrazzano-system vmi-system-es-data-2
-kubectl rollout status deployment -n verrazzano-system vmi-system-es-data-2 -w
- 
-kubectl rollout restart deployment -n verrazzano-system vmi-system-kiali
-kubectl rollout status deployment -n verrazzano-system vmi-system-kiali -w
- 
-kubectl rollout restart deployment -n verrazzano-system vmi-system-grafana
-kubectl rollout status deployment -n verrazzano-system vmi-system-grafana -w
- 
-kubectl rollout restart deployment -n verrazzano-system verrazzano-authproxy
-kubectl rollout status deployment -n verrazzano-system verrazzano-authproxy -w
- 
-kubectl rollout restart daemonset -n verrazzano-system fluentd
-kubectl rollout status daemonset -n verrazzano-system fluentd -w
- 
-kubectl rollout restart deployment -n verrazzano-system verrazzano-console
-kubectl rollout status deployment -n verrazzano-system verrazzano-console -w
- 
-kubectl rollout restart deployment -n verrazzano-system weblogic-operator
-kubectl rollout status deployment -n verrazzano-system weblogic-operator -w
-```
+Follow these [instructions](./upgrade-istio.md) to upgrade to Istio 1.19.9.
```

Comment:
We are not consistent with these links.  I think it is goo to have
"See [Upgrade to Istio 1.19.9](..). or something more descriptive.

Reply:
I pushed that change.